doc/source/ScriptingAPI/DictionarySyntax/index.md

```diff
@@ -0,0 +1,65 @@
+# Dictionary Syntax #
+
+Gaffer scripts use Python's dictionary syntax and notation to define the node graph. As you add to and manipulate the node graph, the dictionary builds a compendium of instances of node and plug classes. Each node is a dictionary key, with a node class as its value. Each plug is a sub-key of a node's key, with a plug class as its value. The following pseudocode shows a rough approximation of a node graph's dictionary:
```

Comment:
"Gaffer's Python API allows you to use Python's dictionary syntax and notion to access the node graph" perhaps?  The node graph is serialized to Python and accessed through Python, but not defined in Python.  Perhaps this is pretty pedantic for a new Gaffer user?

Reply:
"Each node is a dictionary key" -> "Each node can be accessed by using it's name as a dictionary key in its parent"?  Hmm, unwieldy, but it feels wrong to say a node is a dictionary key.